src/pykeen/models/base.py

```diff
@@ -2,15 +2,16 @@
 
 """Base module for all KGE models."""
 
+from __future__ import annotations
```

Comment:
this is for backward compatibility with Python 3.6?

Reply:
no this is python 3.7+. it allows you to use a class in a type annotation of a function inside that class

Ref: https://www.python.org/dev/peps/pep-0563/